.prettierignore

```diff
@@ -0,0 +1 @@
+./src/index.ts
```

Comment:
Why we need this?

Reply:
Removed. 

Prettier was formatting in a way that caused a bug in this file, however, this has been fixed was some configurations.

Reply:
OK. Nice.